Добавил описание понятия "веха"

Danil-Prokofiev opened the pull request:
-Добавил описание понятию _вехи_ в файле 'issues.md' в каталоге 'Student-timetable/Docs/'.
-Добавил ссылку на сайт с полной информацией о _вехах_ 
-Добавил ссылку на сайт с описание создания _вехи_

Danil-Prokofiev committed:
Added a description to the level 3 "milestones" header.

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,19 @@
+-Срок выполнения вехи
+-Процент завершения вехи
+-Количество открытых и закрытых 'issues' и 'pull request'’ов, связанных с вехой
+-Список открытых и закрытых 'issues' и 'pull request'’ов, связанных с вехой
```
немного не правильно использован синтаксис для ненумерованного списка

YuriSilenok replied:
если хотите что-то выделить, то лучше использовать `
ознакомьтесь с синтаксисом [здесь](https://guides.github.com/features/mastering-markdown/)

Danil-Prokofiev replied:
Изменил синтаксис ненумерованного списка согласно Mastering Markdown. Ссылка на коммит: https://github.com/Students-of-the-city-of-Kostroma/Student-timetable/pull/1432/commits/a481032a9c2c4e1734b53bb0edb16f94e41c92ed

Danil-Prokofiev committed:
Изменил синтаксис ненумерованного списка

Заменил символ '-' на '*'. Данный символ является синтаксически правильным для ненумерованного списка по Mistering Markdown. Добавил пробел между символом и текстовой строкой

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,19 @@
 ### Проекты
 ### Вехи
+Вехи можно использовать для отслеживания прогресса в работе над несколькими 'issues' или 'pull requests'’ами в репозитории. 
+Когда вы создаете веху, вы можете ее «связать» с 'issues' и 'pull request'’ами.
```
пожалуйста, для выделения слов используйте символ `, так принято во все документации. Эти требования так же перечислены в [документации проекта](https://github.com/Students-of-the-city-of-Kostroma/Student-timetable/tree/dev/Docs/Contributers/DocStyle).

YuriSilenok reviewed (changes requested):
Так же хотелось видеть разъяснения по поводу того, как использовать вехи. Нужно расписать как они используются для обозначения спринтов, что нужно писать в описание вехи. Для этого я рекомендую заглянуть в закрытые вехи этого проекта.

Danil-Prokofiev committed:
Изменил символ для выделения текста ' на ` 

Изменил символ ' на ` в соответствиями с требованиями стилистического оформления документов. Требования к стилистическому оформлению можно найти по следующей ссылке: https://github.com/Students-of-the-city-of-Kostroma/Student-timetable/tree/dev/Docs/Contributers/DocStyle

Danil-Prokofiev committed:
Добавил информацию о понятии веха

* Расписал как вехи используются для спринтов
* Добавил информацию о "подвязывании" `issues`
* Дал ссылку на официальную документацию GitHub о связывании `issues`, где наглядно показано как выполнять связывание. Это позволит лучше понять материал
* Вывел все строки с ссылками на источники как цитаты, чтобы их было визуально проще отделить от основного материала.

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,34 @@
 Метки необходимы для классификации задачи. Задача которая имеет правильные метки легко ищется. Задача без меток не может существовать. Более подробные требования к меткам Вы можете ознакомится [здесь](/Docs/labels.md)
 ### Проекты
 ### Вехи
+Вехи можно использовать для отслеживания прогресса в работе над несколькими `issues` или `pull requests`’ами в репозитории. 
```
Лишний апостроф перед `ами в репозитории.`

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,34 @@
 ### Проекты
 ### Вехи
+Вехи можно использовать для отслеживания прогресса в работе над несколькими `issues` или `pull requests`’ами в репозитории. 
+Когда вы создаете веху, вы можете ее «связать» с `issues` и `pull request`’ами.
```
Лишний апостроф перед `ами.`

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,34 @@
+* Сделанное пользователем описание вехи, которое может включать в себя такие сведения как: описание проекта, связанные с вехой команды и прогнозируемые сроки выполнения
+* Срок выполнения вехи
+* Процент завершения вехи
+* Количество открытых и закрытых `issues` и `pull request`’ов, связанных с вехой
```
’ов, связанных с вехой

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,34 @@
+* Срок выполнения вехи
+* Процент завершения вехи
+* Количество открытых и закрытых `issues` и `pull request`’ов, связанных с вехой
+* Список открытых и закрытых `issues` и `pull request`’ов, связанных с вехой
```
’ов, связанных с вехой

YuriSilenok reviewed (changes requested):
в целом хорошо, осталось исправить указанные опечатки

Danil-Prokofiev committed:
Убрал все лишние апострафы

* Убрал апостроф перед `ами в репозитории.` в первом абзаце
* Убрал апостроф перед `ами.` в первом абзаце
* Убрал апостроф перед  `ов, связанных с вехой` в двух последних элементах ненумерованного списка во втором абзаце
* Убрал апостроф перед `а` в первом элементе ненумерованного списка о проценте завершения вехи

YuriSilenok commented on Docs/issues.md:
```diff
@@ -7,6 +7,34 @@
+
+> Больше информации о вехах можно узнать [здесь](https://docs.github.com/en/github/managing-your-work-on-github/tracking-the-progress-of-your-work-with-milestones/about-milestones)
+
+> О том как создать веху можно узнать [здесь](https://docs.github.com/en/github/managing-your-work-on-github/tracking-the-progress-of-your-work-with-milestones/creating-and-editing-milestones-for-issues-and-pull-requests)
```
символ `>` используется для цитирования. здесь вы не цитируете, а предлагаете некоторый набор полезных ссылок.

Danil-Prokofiev committed:
Изменил формат представления полезных ссылок

* Убрал символ `>` для цитирования. 
* Добавил строчку `Полезные ссылки:`
* Сделал все строки с полезными ссылками элементами ненумерованного списка
